### DISCUSSION TITLE
New Release forecast to incorporate recent fixes, especially #153?

### DISCUSSION THREAD

**Discussion by aGrimRepoMan** [octoshrimpy/quik#158] [Q&A]
Hi @octoshrimpy ,

I'm enjoying QUIK.   Thanks very much for your work in keeping it alive.

I see the fix for #153 likely resolves an issue I'm having. Would it be possible for you to release a new build to F-Droid which contains that fix?  If so, roughly when is that forecast?  [I'm guessing it requires a fair bit of testing, so I can certainly understand it might take some time].

Thanks again!

**Comment by octoshrimpy** [comment]
Hi! I'm currently mid-move, but have been cleaning up errors and refactoring relating to that issue. that and MMS compression sometimes not working are my top priorities.

**Comment by aGrimRepoMan** [comment]
Understand. Thanks for the update and for your efforts on this app.
